components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -449,7 +449,7 @@ const AddToListDropdown = ({ username }: { username: string }) => {
       open={selectListOpen}
       setOpen={setSelectListOpen}
       emptyState={
-        <div className="">
+        <div className="mt-1 mr-2 px-6 py-2">
```

Comment:
Just out of curiosity, why did not we have `p-1 m-1` directly?

Reply:
Thanks @a0m0rajab and @nickytonline for the suggestions. I have made the changes accordingly.